modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/transcripts/TranscriptPage.kt

```diff
@@ -145,19 +153,37 @@ private fun TranscriptContent(
                 html = displayString.toString(),
                 color = colors.textColor(),
                 textStyleResId = UR.style.H40,
+                selectable = true,
                 modifier = modifier
                     .padding(horizontal = 16.dp)
                     .padding(bottom = bottomPadding)
                     .verticalScroll(rememberScrollState()),
             )
         } else {
-            Text(
-                displayString,
-                modifier = Modifier
-                    .padding(horizontal = 16.dp)
-                    .padding(bottom = bottomPadding)
-                    .verticalScroll(rememberScrollState()),
-            )
+            val customMenu = buildList {
+                if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
+                    add(CustomMenuItemOption.Share)
+                }
+                add(CustomMenuItemOption.SelectAll)
```

Comment:
I have not added `Share` custom menu for newer devices (API level >33) as copy option provides sharing once the text is selected. 

Note that transcripts with HTML mime-type use Android view to render HTML content and the text selection toolbar displays a share option even on newer devices. But I plan to replace `Android View` with `Compose View` once Compose `1.7.0` is stable that can parse HTML ([ref](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.7.0-alpha06:~:text=Added%20parseAsHtml%20method%20for%20styled%20strings)).